embedded_hub.go

```diff
@@ -22,11 +17,17 @@ import (
 	"sync"
 	"time"
 	"unsafe"
+
+	"github.com/enriquebris/goconcurrentqueue"
+	"github.com/juju/loggo"
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 var localPeer *peer_service.PeerService
 
-var log = loggo.GetLogger("embedded_hub")
+// var log = loggo.GetLogger("embedded_hub")
```

Comment:
Why is this line commented?

Reply:
![image](https://user-images.githubusercontent.com/21281457/151544194-f8b69e40-5334-4684-b1f1-492db598ec3f.png)
I was getting this error. So I commented it

Reply:
You should not compile main and embedded_hub simultaneously,  this are different modules, Don't use main module and don't add it to build please

Reply:
got it.